EasyPost/Models/API/Options.cs

```diff
@@ -116,8 +116,10 @@ public class Options : EasyPostObject
 
         [JsonProperty("carrier_insurance_amount")]
         public string? CarrierInsuranceAmount { get; set; }
```

Comment:
question: can we globally ignore the comment requirements in just this file? That would clean up a bunch of these warnings on GitHub

Reply:
Yes, I plan to make adjustments to the style guide for it.